C++.en.md

```diff
@@ -0,0 +1,146 @@
+Wiren Board C++ Style Guide
+========================
+
+The following sheet is based on Yandex code style for C++. If any guideline is missing from the following document, 
+please follow Yandex official documentation: https://clickhouse.yandex/docs/en/development/style.html . But if you
+see any collision between documents, please follow the guide below. 
+
+## Marking
+
+For identation blocks only spaces are used and 4 spaces count as 1 tab.
+
+Aritmetic operations are always separated by spaces:
+```C++
+int sum = a + b;
+```
+Whitspace should be placed before each bracket opening.
+```C++
+int function (int a, int b, int c) 
+{
+  int d = abs ( (a * sqrt (b)) + c);
```

Comment:
the example already breaks the rule above :) Better remove the rule at all.

Reply:
Ok